pages/common/git-switch.md

```diff
@@ -0,0 +1,25 @@
+# git switch
+
+> Switch between git branches. Requires git version 2.23+.
+> See also `git checkout`.
+> More information: <https://git-scm.com/docs/git-switch/>.
+
+- Switch to an existing branch:
+
+`git switch {{branch_name}}`
+
+- Create a new branch and switch to it:
+
+`git switch --create {{branch_name}}`
+
+- Create a new branch based on an existing commit:
+
+`git switch --create {{branch_name}} {{start_point}}`
+
+- Also update all submodules:
```

Comment:
This sounds somewhat informal. I am wondering if it sounds any better if we drop "Also" ?